Code/Framework/AzQtComponents/AzQtComponents/Utilities/ScreenGrabber_win.cpp

```diff
@@ -29,7 +29,7 @@ namespace AzQtComponents
 
     namespace
     {
-        std::string GetLastErrorString()
+        [[maybe_unused]] std::string GetLastErrorString()
```

Comment:
This is fine, but you can squeeze a little bit more build optimization be wrapping this function inside a `#if defined(AZ_ENABLE_TRACING)` block.